### ISSUE TITLE
Desktop Interface: Canvassing Record Input Interface

### ISSUE THREAD

**Issue by aarontitus** [aarontitus/crisiscleanup-legacy#116]
_Original author: v...@aarontitus.net (January 01, 2013 23:09:01)_

The Desktop Canvassing Record Input Interface should be similar to the Assessment/Intake/Work Order Input Interface, but much simpler. See #114. The interface should include:
- Address (Required). This should work exactly as the Assessment/Intake/Work Order Input Interface.
- Lat/Long (Required). This should work exactly as the Assessment/Intake/Work Order Input Interface.
- Status (Required). Include the following Buttons or check-boxes (E.g. &quot;No Needs: Helped by Others&quot;), or nested options (e.g. &quot;No Needs&quot; -&gt; &quot;Helped by Others&quot;).  Valid Statuses (or their aliases) include:
  - No Needs: No Help Wanted
  - No Needs: Unaffected
  - No Needs: Helped by Others
  - No Needs: Rejected (Please Explain)
  - No Needs: Needs are Out of Scope
  - Follow Up: Nobody Home
  - Follow Up: Owner Not Present
  - Follow Up: May Need Help in the Future
  - Follow Up: Needs Help, but Not Ready
  - Needs Help/ Assessed [If this selection is chosen in the interface, then the user is forwarded directly to the Assessment/ Intake/ Work Order Input Form, before creating a Canvassing Record. After the assessment is complete, a Work Order record is created, and also Canvassing Record at the same time]
- Status Notes (Optional). A text field with additional explanation of the status, if desired.
- Name (Optional). A text field
- Phone Number (Optional)

The Desktop Canvassing Record Input Interface should contain a link to the Desktop Canvassing Map (See #115).

Note: #114 must be completed first.

_Original issue: http://code.google.com/p/sandy-disaster-recovery/issues/detail?id=116_
